modules/pekko-http/README.md

```diff
@@ -0,0 +1,36 @@
+# Pekko-Http module for PureConfig
+
+Adds support for [Pekko-Http](https://pekko.apache.org/docs/pekko-http/current/common/http-model.html)'s Uri class to PureConfig. PRs adding support
+for other classes are welcome :)
+
+## Add pureconfig-pekko-http to your project
+
+In addition to [core PureConfig](https://github.com/pureconfig/pureconfig), you'll need:
+
+```scala
+libraryDependencies += "com.github.pureconfig" %% "pureconfig-pekko-http" % "@VERSION@"
```

Comment:
Running `pekko-http-docs/mdoc` should have this replaced.

```suggestion
libraryDependencies += "com.github.pureconfig" %% "pureconfig-pekko-http" % "0.17.5"
```

Reply:
My bad, I copy pasted those lines and forgot to run the command.
Regenerated the files in 8655d5b